README.md

```diff
@@ -85,7 +94,7 @@ github "hyperoslo/Tabby"
 
 ## Author
 
-Made by Hyper Interaktiv AS. Contact us at ios@hyper.no.
+Made by Hyper Oslo. Contact us at ios@hyper.no.
```

Comment:
Yeah that would be nice. Hyper Interaktiv AS is fine I guess. But I think that's old.